versions.json

```diff
@@ -2,11 +2,17 @@
   "notice": "This file is not maintained outside of the main branch and should only be used for tooling.",
   "versions": [
     {
-      "version": "8.10.0",
+      "version": "8.11.0",
       "branch": "main",
       "currentMajor": true,
       "currentMinor": true
     },
+    {
+      "version": "8.10.0",
+      "branch": "8.10",
+      "currentMajor": true,
+      "previousMinor": true
+    },
     {
       "version": "8.9.1",
       "branch": "8.9",
```

Comment:
`previousMinor: false` maybe.  Don't feel strongly about it, either way works for me.